### DISCUSSION TITLE
Network Manager service error

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#10499] [2.4]
Hello Security Onion folks,

I recently pulled down 2.4.2 iso and attempted to install in my proxmox lab.

When running network configuration step I noticed it was unable to complete due to an issue with Network Manager service not starting properly. Has anyone else experienced this since release?

Cole

**Comment by dougburks** [comment]
I don't believe we've seen this issue in our automated or manual testing. In fact, the screenshots at https://docs.securityonion.net/en/2.4/first-time-users.html were generated using the 2.4.2 ISO image in Proxmox.

Did you verify the ISO image as shown at https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/VERIFY_ISO.md?

Are you able to duplicate this issue on a fresh installation?

Are you able to provide any more details?

**Comment by ColeVan** [comment]
Hello Doug,

Correct, I verified the iso image before downloading. I will pull down a fresh ISO now and try to replicate the issue.

Here are a couple of screenshots regarding the error I encountered. Thoughts?

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/488284bb-997a-44ff-92a0-3e514fbb3052)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/4d996758-32d5-44cc-ae70-cf6bb75c69f0)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/db7950f6-eaf6-4a13-9c3a-bbbaeb67b176)

**Reply by dougburks** [reply]
The first line shows `Read-only file system`. That would definitely cause issues with Setup. Did you have some kind of ungraceful shutdown or something else that might have caused file system issues? 

In any case, I suspect that a fresh installation on a fresh file system should work better.

**Reply by ColeVan** [reply]
The setup was business as usual. Nothing too out of the ordinary. Maybe a fluke?? Downloading the new ISO now. I'll report back later today on the status. Thanks Doug!!

**Reply by ColeVan** [reply]
New iso downloaded.

Just ran through the pre-installation and rebooted the system. Now I'm seeing a totally different error. Upon creating the Admin user "hunter" during pre-installation there were no issues. When I rebooted the system upon completion I expected to be thrown right into the setup process as usual. Unfortunately, I was not. I then tried to run the setup and I got an error stating the "hunter" user is not in the sudoers group which is very odd. Also, I know for a fact that both interfaces are up but looking at the terminal they are still in a down state.

Thoughts?


![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/b02d59af-beef-481c-b725-21f332ec323f)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/44d9c38e-53a8-45f9-8032-a4bf37374920)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/1e546974-98d1-4221-808e-1c865a9899f0)

**Reply by dougburks** [reply]
I just ran through an installation and specified `hunter` as my username. After installation, I logged in as `hunter` and got the Setup screen as expected confirming that the sudoers config is correct:
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/fbfd50ed-e732-4d53-a6d6-0b0ffc7f0928)

Just to double-check, what is the md5sum of your ISO image?

**Reply by ColeVan** [reply]
Ok, ran through the setup one more time. The only thing I did differently was deselected iso from the boot order when restarting the instance. Here are my settings, everything is running smoothly so far. Maybe just a weird bug with proxmox somewhere.
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/16ff3de8-bc5c-4aa5-97d9-0fbf7473c0f1)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/14e36813-3d28-4524-a491-fc1566d4a22d)

**Reply by ColeVan** [reply]
eb861efb7f7da6fb418075b4c452e4eb
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/70167373/6bd60c4a-b303-41dd-99e9-90880b81734f)

**Reply by dougburks** [reply]
md5sum is correct. Glad it's working now.

**Reply by ColeVan** [reply]
So many settings in proxmox. Most likely a user error on my end. Thanks, Doug!!
